adcircpy/_fort15.py

```diff
@@ -2,9 +2,8 @@
 from functools import lru_cache
 import pathlib
 
-import numpy as np
-
 from adcircpy.forcing.tides.tpxo import TPXO
+import numpy as np
```

Comment:
Why move it down here? Usually the third party import go before the local module imports

Reply:
This is the only remaining point that has me confused

Reply:
I think that's an artifact of a merge conflict, just pushed the autoformat that orders it correctly